src/Pages/Home.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import FirstFoldAboutKaran from '../Sections/FirstFoldAboutKaran';
 import Footer from '../Sections/Footer';
+import OrgzitWorkEx from '../Sections/OrgzitWorkEx';
+import TCSWorkEx from '../Sections/TCSWorkEx';
 
 class Home extends React.Component {
    render() {
       return (
          <div>
             <FirstFoldAboutKaran />
+            <OrgzitWorkEx />
```

Comment:
I think work experience should be a reusable component and we can pass props about the company rather than using different components for different companies.

Another thing is, making these separate components does not make perfect sense because you won't be using it anywhere else.